### ISSUE TITLE
screen issues on iPhone and android

### ISSUE THREAD

**Issue by janeatcc** [creativecommons/cccatalog-frontend#377]
"Search" and "Filter" don't fit here:
![photo_2019-05-03_22-28-49](https://user-images.githubusercontent.com/6394777/57164487-1c1bdf00-6da9-11e9-9084-613bc80f364a.jpg)

If I put the device in landscape mode I can't use the site:
![photo_2019-05-03_22-28-54](https://user-images.githubusercontent.com/6394777/57164494-23db8380-6da9-11e9-8a39-acb7971a7eca.jpg)

If I scroll down in landscape mode the header disappears and I see a little piece of the website but it's not enough to use the website. I think that the black footer it's fixed and should not be fixed (it covers the content):
![photo_2019-05-03_22-28-57](https://user-images.githubusercontent.com/6394777/57164500-29d16480-6da9-11e9-88b3-5fac3f7482f6.jpg)

**Comment by elharony** [comment]
Hi @janeatcc ,

I've fixed the `footer/homepage` responsive problem in #379 and I will check the `Search/Filter` today as well.

**Comment by elharony** [comment]
Hi @janeatcc ,

Thanks for reporting those critical UI problems, both issues are solved in #379 & #380! 🙆‍♂
